lib/phobos/cli.rb

```diff
@@ -49,6 +49,13 @@ def init
              aliases: ['-b'],
              banner: 'File path to load application specific code',
              default: 'phobos_boot.rb'
+      option :listeners,
+             aliases: ['-l'],
+             banner: 'Listeners config file'
```

Comment:
Since you can still define the listeners in the config file it would be nice to add more information here, maybe saying `Separate listeners config file (optional)`. WDYT?